cmd/plugin/plugin.go

```diff
@@ -29,39 +29,88 @@ func NewCommand(out output.Output) *cobra.Command {
 
 // InstallCommand creates a new command to install
 func InstallCommand(out output.Output) *cobra.Command {
-	return &cobra.Command{
+	var (
+		pluginSource  string
+		pluginName    string
+		pluginVersion string
+		pluginURL     string
+		avm           avmpkg.AVM
+	)
+
+	cmd := &cobra.Command{
 		Use:   "install",
 		Short: "Installs a tool for a specific plugin",
-		RunE: func(cmd *cobra.Command, args []string) error {
-			avm, err := avmpkg.New(out)
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if pluginSource != "asdf" {
+				return fmt.Errorf("We do not support any source other than asdf for now, got: %s", pluginSource)
+			}
+
+			if pluginName == "" {
+				return fmt.Errorf("Please provide a plugin name")
+			}
```

Comment:
Do we need this since the name is marked as required?

Reply:
You could run the tool with `--name ""` then the error message would be: `plugin  not found`        
I will remove it.